README.md

```diff
@@ -29,7 +29,16 @@ Navigate to `localhost:8080/components/my-element/demo.html`
   * `-b <browsername>` use this browser instead of default (ex: 'Google Chrome Canary')
   * `-H <hostname>` use this hostname instead of localhost
 
-## Run Tests
+## Contributing
+
+```
+git clone git@github.com:polymerlabs/polyserve.git
+cd polyserve/
+npm install
+npm run build
```

Comment:
Oh yes definitely. It was not difficult to write TS, building it took me a couple of minutes to figure out by manually inspecting the `package.json`. Also updated the other command line references to the CLI format.